detectable/src/main/java/com/synopsys/integration/detectable/detectables/yarn/ExternalIdDependencyGraphBuilder.java

```diff
@@ -0,0 +1,216 @@
+package com.synopsys.integration.detectable.detectables.yarn;
+
+import com.synopsys.integration.bdio.graph.BasicDependencyGraph;
+import com.synopsys.integration.bdio.graph.builder.LazyBuilderMissingExternalIdHandler;
+import com.synopsys.integration.bdio.graph.builder.LazyExternalIdDependencyGraphBuilder;
+import com.synopsys.integration.bdio.graph.builder.LazyId;
+import com.synopsys.integration.bdio.graph.builder.MissingExternalIdException;
+import com.synopsys.integration.bdio.model.dependency.Dependency;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class ExternalIdDependencyGraphBuilder extends LazyExternalIdDependencyGraphBuilder {
```

Comment:
It's somewhat simple to release a new version of integration-bdio if we did want to contribute this there. Not worth holding up the release for of course.

Reply:
Also, the additional effort to test. We can consider to do so after this release.